src/control/cmd/daos_server/storage.go

```diff
@@ -129,30 +130,46 @@ func (cmd *storageScanCmd) Execute(args []string) error {
 
 	cmd.log.Info("Scanning locally-attached storage...")
 
+	var bld strings.Builder
 	scanErrors := make([]error, 0, 2)
 
-	res, err := svc.NvmeScan()
+	nvmeResp, err := svc.NvmeScan(bdev.ScanRequest{})
 	if err != nil {
 		scanErrors = append(scanErrors, err)
 	} else {
-		ctrlrs := proto.NvmeControllers{}
-		if err := ctrlrs.FromNative(res.Controllers); err != nil {
-			scanErrors = append(scanErrors, err)
-		} else {
-			cmd.log.Info(ctrlrs.String())
+		_, err := fmt.Fprintf(&bld, "\n")
+		if err != nil {
+			return err
+		}
+		if err := control.PrintNvmeControllers(nvmeResp.Controllers, &bld); err != nil {
```

Comment:
it's not just this change that requires them to be exported, it's not much time invested to improve the consistency even if they are going to be removed. I'm imagining there may be a pushback to removing them so it may be that they remain for a little while.

Reply:
This is a temporary move to give consistency between storage commands on different binaries (dmg/daos_server) and will be reverted in during work on DAOS-5521 when removing all storage lib dependencies from daos_server binary (force forwarding to the privileged binary and remove storage subcommands from daos_server).  DAOS-5521 won't necessarily be addressed in the short-term given its priority relative to other scheduled tasks and therefore I'm of the opinion that command output should be made consistent in the interim. This PR enforces the use of a single print function for NVMe structures and removes duplication.

Reply:
The point of putting the printers in their own package (dmg/pretty) was to make them importable from outside of dmg. It looks like a sub-package, but Go doesn't have this concept. Each package is an independent namespace, not part of a hierarchy. If @kjacque is OK with it, let's plan on moving these back into dmg/pretty in your next PR.

Reply:
Works for me.